### PR TITLE
Improve English Translations

### PR DESCRIPTION
**Pull request by MMK21Hub**
- Better coverage for `en-pirate`
- Fix capitalisation in `en` and `owo`
- Reword a few strings to make more sense
- Grammar fixes

### CODE REVIEW

**Comment by overlisted** [comment]
also consider adding these changes to the owo language

File: kofta/public/locales/en/translation.json [hunk 1]
```diff
@@ -189,28 +189,28 @@
...
-        "makeRoomCreator": "make room admin"
+        "blockUser": "Block user",
+        "makeMod": "Promote to Mod",
+        "unmod": "Demote from Mod",
+        "addAsSpeaker": "Add as Speaker",
+        "moveToListener": "Move to Listener",
+        "banFromChat": "Ban from Chat",
+        "banFromRoom": "Ban from Room",
+        "goBackToListener": "Go Back to Listener",
+        "deleteMessage": "",
```
Comment (benawad) [inline kofta/public/locales/en/translation.json]: Any reason you left this blank?

**Review by benawad — CHANGES_REQUESTED** [review]

File: kofta/public/locales/en/translation.json (hunk 1 above)
Comment (MMK21Hub) [reply kofta/public/locales/en/translation.json]: Didn't mean to do that; I'll fix it.

Comment (MMK21Hub) [reply kofta/public/locales/en/translation.json]: Resolved by 225e2f6

**Comment by MMK21Hub** [comment]
> also consider adding these changes to the owo language

Marked this as a draft to let me do this

File: kofta/public/locales/en-pirate/translation.json [hunk 2]
```diff
@@ -53,17 +53,17 @@
       "featureText_4": "Open Source",
       "featureText_5": "Text Chat",
       "featureText_6": "Powered by Doge",
-      "loginGithub": "login with GitHub",
-      "loginTwitter": "login with Twitter",
-      "createTestUser": "create test user"
+      "loginGithub": "Set Sail wit' with GitHub",
```
Comment (noftaly) [inline kofta/public/locales/en-pirate/translation.json]:
```suggestion
      "loginGithub": "Set Sail wit' GitHub",
```

**Comment by amitojsingh366** [comment]
whats the status on this @MMK21Hub?

**Comment by MMK21Hub** [comment]
> whats the status on this MMK21Hub?

@amitojsingh366 It's been on my to-do list for a while, I'm planning to work on it today and tomorrow. Should be ready to merge by then.

File: kofta/public/locales/en-pirate/translation.json (hunk 2 above)
Comment (MMK21Hub) [reply kofta/public/locales/en-pirate/translation.json]: Thanks for spotting that mistake!

**Comment by MMK21Hub** [comment]
> also consider adding these changes to the owo language

Done with 8975a3f

**Pull request merged**